data/mods/DinoMod/overmap/overmap_terrain_zoo.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "type": "city_building",
-    "id": "dinozoo",
+    "id": "DinoZoo",
```

Comment:
`id` should not be changed

Reply:
Good point, I didn't notice, thanks for double checking! Let me fix it quick